home/ivp-planning/advanced-tools/the-room-tool.md

```diff
@@ -8,17 +8,14 @@ The room tool is a special variant of the [path tool](path-tool.md). We recommen
 
 ## Creating a room:
 
-Rooms are based on measurements which can be created with the [path tool](path-tool.md). Afterwards, every path can be turned into a room by **clicking on the "Room" tab** of the [path panel](../user-interface/path-panel.md) that is displayed as soon as a path is [selected](path-tool.md#path-selection-and-editing).
+Rooms are based on measurements which can be created with the [path tool](path-tool.md). Afterwards, every path can be turned into a room by **changing the type** in the Measurements tab of the [properties](../user-interface/the-info-panel.md) that is displayed as soon as a path is [selected](path-tool.md#path-selection-and-editing).
```

Comment:
This isn't a tab anymore. It's now using a foldout

Reply:
changed.